docs/apache-airflow/redirects.txt

```diff
@@ -19,8 +19,8 @@
 
 administration-and-deployment/security/index.rst security/index.rst
 administration-and-deployment/security/kerberos.rst security/kerberos.rst
-administration-and-deployment/security/access-control.rst ../../apache-airflow-providers-fab/stable/auth-manager/access-control.rst
-administration-and-deployment/security/access-control/index.rst ../../apache-airflow-providers-fab/stable/auth-manager/access-control.rst
+administration-and-deployment/security/access-control.rst ../apache-airflow-providers-fab/stable/auth-manager/access-control.rst
+administration-and-deployment/security/access-control/index.rst ../apache-airflow-providers-fab/stable/auth-manager/access-control.rst
```

Comment:
There are two more links below that need fixing as well.

BTW. The `../` rule here is ratehr misleading (though I can imagine where it came from).

Maybe copy the explanation from K8S executor redirects below?


```
# Moving provider executor docs to providers
## The ``../`` indicates that it will move to the root of the docs directory, unlike the rest of
## the entries here which move to docs/apache-airflow.
## It's okay to include ``/stable/``, because there's no relationship between a version of
## Airflow and the version of any provider package.
```

Reply:
Good point

Reply:
Done